### DISCUSSION TITLE
Alternative For "Mint a Collectable" in Phantom Wallet...Help to code alternative SDK to "MINT AN ITEM":

### DISCUSSION THREAD

**Discussion by virginiabrobertson** [phantom/docs#250] [General]
**Similar to "Mint a Collectable" Help to code an alternative SDK:**
Labels for metadata: 
Token ID = "ITEM"  (This would always be fixed in this SDK)
Picture required
Additional Required Metadata with labels:
What (In place of NAME)
Description (optional in place of DESCRIPTION)
Where (additional attribute)
What Price (additional attribute)
**So what is the most direct path to make this SDK available on Iphone and Android for Phantom wallet?**

**Comment by adamdelphantom** [comment]
Hello @virginiabrobertson,

Thank you for writing in! Phantom does not offer a traditional sdk. For mobile you can interact either via the [Phantom provider](https://docs.phantom.app/solana/detecting-the-provider) when users are visiting your dapp in the Phantom dapp browser.

If you are building a mobile app you will want to make use of [deeplinks](https://docs.phantom.app/phantom-deeplinks/deeplinks-ios-and-android) for connecting and signing any transactions.

For your example the general workflow would be that users could populate the fields on your app then you would create a transaction with the relevant data for minting and send the transaction signing request to Phantom either via the provider or deeplinks. From there your user would be prompted to confirm or deny the signing request.

**Reply by virginiabrobertson** [reply]
Ok ademdel thank you and I appreciate you. I'll investigate.
vr

**Reply by virginiabrobertson** [reply]
So when an NFT is auto-created in the Phantom mobile app, why is it not searchable by its name in solana explorers?

**Reply by virginiabrobertson** [reply]
...and why does the NFT created not even show up if I am looking at the NFT tokens in a specific wallet that I created it in?
